cmd/ncproxy/hcn_networking_test.go

```diff
@@ -941,7 +946,7 @@ func TestAddEndpoint_NoError(t *testing.T) {
 	gService := newGRPCService(agentCache, networkingStore)
 
 	// create test network namespace
-	namespace := hcn.NewNamespace(hcn.NamespaceTypeHostDefault)
+	namespace := hcn.NewNamespace(hcn.NamespaceTypeHost)
```

Comment:
Why did we change this?

Reply:
im not sure, but we either get it (if it exists) or create a host default one below (L958-970)
we need to have a host default and separate host one to test the `attachToHost` functionality, which attaches to the host default namespace

Reply:
ill update the comments to make that more clear